resources/lib/SubtitleHelper.py

```diff
@@ -33,3 +33,23 @@ def convert_to_utf(file):
             output.write(srt_data)
     except UnicodeDecodeError:
         log(__name__, "got unicode decode error with reading subtitle data")
+
+def normalizeString(str):
+    return unicodedata.normalize(
+        'NFKD', unicode(unicode(str, 'utf-8'))
+    ).encode('utf-8', 'ignore')
+
+def check_and_parse_if_title_is_TVshow(manualTitle):
```

Comment:
I totally changed the implemention and now it's using regex.
It looks a little longer, but with no bugs for sure.